### PR TITLE
Add Related Activities Section

### PR DESCRIPTION
**Pull request by lananhkn**
Please review my code.

### CODE REVIEW

File: VMS/Pages/Homepage/RelatedActivities.razor [hunk 1]
```diff
@@ -0,0 +1,140 @@
+﻿@attribute [Route(Routes.Activities)]
```
Comment (LongDuongBIT) [inline VMS/Pages/Homepage/RelatedActivities.razor:1]: Is this a Blazor page???
If not, please remove this attribute.

File: VMS/Pages/Homepage/RelatedActivities.razor.scss [hunk 2]
```diff
@@ -0,0 +1,189 @@
+﻿@import '../../Resources/scss/common/_variables.scss';
+
+$greyicon: #B4B4B4;
+$grey: #F3F3F5;
+$blue-hv: #0B588c;
+$green-hv: #306C14;
+$grey-hv: #646464;
```
Comment (LongDuongBIT) [inline VMS/Pages/Homepage/RelatedActivities.razor.scss:7]: Can you explain why you don't move those color variables into **_variables.scss**???

**Review by LongDuongBIT — COMMENTED** [review]
@lananhkn I have some comments, please take a look at them.

File: VMS/Resources/src/homepage/index.js [hunk 3]
```diff
@@ -48,4 +48,20 @@ const playVideo = (src) => {
...
\ No newline at end of file
+const getUserLocation = () => {
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            return null;
+        }
+    }
+
+function showPosition(position) {
```
Comment (LongDuongBIT) [inline VMS/Resources/src/homepage/index.js:59]: Please use ES6 syntax for consistency.

**Comment by lananhkn** [comment]
> @lananhkn I have some comments, please take a look at them.
I fixed all of these things, please review them again.

**Review by LongDuongBIT — APPROVED** [review]

**Pull request merged**